core/client/scenes/scene-editor.js

```diff
@@ -4,6 +4,38 @@ import { canEditActiveLevel } from '../../lib/misc';
 
 const editorGraphicsDepth = 10002;
 
+const previewInfo = {
+  previewTiles: {},
+  lastSelectedTiles: {},
+  lastMousePosition: {},
+  previousTiles: null,
+};
+
+
+function compareMouseMovements(currentPosition, lastMousePosition) {
+  return currentPosition.x === lastMousePosition.x && currentPosition.y === lastMousePosition.y;
```

Comment:
I guess these are integers here. if you have floats it can be a problem and you'll have to do something like :

```
return Math.abs(currentPosition.x - lastMousePosition.x) < threshold && Math.abs(currentPosition.y - lastMousePosition.y) < threshold;
```

Note: threshold being a small value of type 0.0001, Number.EPSILON will cause problems.

Reply:
I have my answer, they are integers so it's fine 🙂